oracle-ebs-122/2.1/modules/ROOT/pages/index.adoc

```diff
@@ -21,10 +21,29 @@ image::oracle-ebs122-architecture.png[Oracle EBS 12.2 - Connector Architecture]
 
 == Before You Begin
 
-To use this connector, you must:
-
-* Be familiar with Oracle E-Business Suite, Anypoint Connectors, Anypoint Studio, Mule concepts, elements in a Mule flow, and global elements
-* Have Oracle EBS login credentials to test your connection to your target resource
+To use this connector, you must have:
+
+* Familiarity with:
```

Comment:
This wording is not quite right. You don't "have familiarity with something," you are familiar with something. To fix this, I suggest creating a separate bullet for this with the following lead-in:

To use this connector, you must be familiar with:"

Then, for the next bullet: "You must also have:"